blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/MultiBranchPipelineImpl.java

```diff
@@ -255,7 +271,14 @@ public Link getLink() {
         @Override
         public MultiBranchPipelineImpl getPipeline(Item item, Reachable parent, BlueOrganization organization) {
             if (item instanceof MultiBranchProject) {
-                return new MultiBranchPipelineImpl(organization, (MultiBranchProject) item);
+                MultiBranchPipelineImpl mbpi = new MultiBranchPipelineImpl(organization, (MultiBranchProject) item);
+                if (item instanceof WorkflowMultiBranchProject) {
+                    WorkflowMultiBranchProject wfmbp = (WorkflowMultiBranchProject)item;
+                    WorkflowBranchProjectFactory wbpf = (WorkflowBranchProjectFactory)wfmbp.getProjectFactory();
```

Comment:
This seems to break "Multibranch with defaults" plugin as its PipelineBranchDefaultsProjectFactory cannot be cast to WorkflowBranchProjectFactory which leads to a ClassCastException.

Reply:
@kshultzCB Can you look at this plz? :)

Reply:
I confirm ... it break Multibranch With Defaults